app/controllers/navigation/Navigator.scala

```diff
@@ -54,16 +54,19 @@ object Navigator {
 
   val standard: PartialFunction[DeclarationFieldCompanion, Mode => Call] = {
     case BorderTransport => controllers.declaration.routes.DepartureTransportController.displayPage
+    case PreviousDocumentsData => controllers.declaration.routes.NatureOfTransactionController.displayPage
```

Comment:
How do we test this works as we intend?

Reply:
The ViewSpec for these pages will test it.